shopify_app.gemspec

```diff
@@ -10,7 +10,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency('rails', '>= 3.1', '< 5.0')
 
-  s.add_runtime_dependency('shopify_api', '~> 3.2.0')
+  s.add_runtime_dependency('shopify_api', '> 3.2.0')
```

Comment:
Yeah, strictly speaking that's probably the best way to specify it. I'll change it